addons/api/addon/generated/models/auth-method.js

```diff
@@ -55,55 +55,239 @@ export default class GeneratedAuthMethodModel extends BaseModel {
   })
   version;
 
-  @attr('string', { readOnly: true, isNestedAttribute: true }) state;
+  // =attributes(oidc, ldap)
 
-  @attr('string', { isNestedAttribute: true }) issuer;
+  @attr('string', {
```

Comment:
Does this need a `for` that includes oidc and ldap?

Reply:
@ZedLi It works as designed, but if I include the `for` option, I would have the refactor the `changeState` method for the `adapterOptions`?

Reply:
Not sure I understand, why is that? It's only for OIDC and LDAP right?

Reply:
@ZedLi Yes, but we delete the nested attributes for `password`,  so my question would be, should it not need a `for` option because the state attribute is for both?

Reply:
Are you saying because `password` also deletes attributes we don't need it? We should still annotate since if we add another auth method in the future that assumption might not hold